packages/components/stepper/src/js/stepper.controller.js

```diff
@@ -29,14 +29,15 @@ export default class {
     );
   }
 
-  addStep(step) {
-    if (!step.position) {
-      this.steps.push(step);
-    } else {
+  addStep(step, index) {
+    // Priority for position attribute
+    if (angular.isNumber(step.position)) {
```

Comment:
```suggestion
    if (Number.isFinite(step.position)) {
```

Reply:
Why ? The one used works

Reply:
I guess it's to avoid use of `angular` variable

Reply:
I know that, but I keep my question.
Currently, the component works with angular.